src/components/SettingsDialog/Components/AvatarSettings.scss

```
@@ -0,0 +1,144 @@
+@import "constants/style";
+
+.avatar-settings {
+  &__avatar {
+    position: relative;
+    margin: 0 auto;
+
+    &-icon {
+      height: 120px;
+      width: 100%;
+      flex-shrink: 0;
+    }
+
+    &-shuffle {
+      position: absolute;
+      cursor: pointer;
+      right: 0;
+      bottom: 0;
+      display: flex;
+      width: 40px;
+      height: 40px;
+      padding: 4px;
+      border: 0;
+      border-radius: 20px;
+      background-color: $color-white;
+      box-shadow: 0 16px 32px rgba($color-lean-lilac, 0.3);
+      align-items: center;
+      justify-content: center;
+
+      &:hover:after {
+        content: "";
+        width: 100%;
+        height: 100%;
+        background: rgba(var(--accent-color-rgb), 0.1);
+        position: absolute;
+        top: 0;
+        left: 0;
+        border-radius: 20px;
+      }
+
+      > svg {
+        width: $icon--large;
+        height: $icon--large;
+      }
+    }
+  }
+
+  &__settings {
+    background-color: $color-white;
+    border-radius: 8px;
+
+    &-wrapper {
+      overflow-y: auto;
+      padding-right: 10px;
+      margin-right: -20px;
+
+      @include scrollbar();
+
+      @media screen and (min-width: 920px) {
+        padding-right: 20px;
+        margin-right: -30px;
+      }
+    }
+
+    &-group {
+      &:last-child {
+        padding: calc($padding--small / 2) 0;
+      }
+
+      &-header {
+        border-radius: 8px;
+        cursor: pointer;
+
+        &:hover {
+          background-color: rgba(var(--accent-color-rgb), 0.1);
+        }
+      }
+
+      &-item {
+        padding: $padding--small 0;
+        width: 90%;
+        margin: 0 auto;
+
+        &:hover {
+          background-color: rgba(var(--accent-color-rgb), 0.1);
+        }
+
+        &-seperator {
+          margin: 0;
+          margin-left: 10%;
+          width: 80%;
+          border: 0;
+          border-top: 0.5px solid $color-light-gray;
+        }
+      }
+
+      &-seperator {
+        margin: 0;
+        margin-left: 5%;
+        width: 90%;
+        border: 0;
+        border-top: 0.5px solid $color-light-gray;
+      }
+    }
+  }
+}
+
+[theme="dark"] {
+  .avatar-settings {
+    &__avatar-shuffle {
+      background-color: $color-dark-mode-note;
```

Würde hier im Darkmode die Farbe vom Shadow ändern, das sieht sonst mMn etwas seltsam aus. Was hältst du hiervon?
```suggestion
      background-color: $color-dark-mode-note;
      box-shadow: 0 16px 32px rgba($color-darkest-gray, 0.8);
```